llama_stack/distribution/datatypes.py

```diff
@@ -117,6 +117,21 @@ class Provider(BaseModel):
     config: Dict[str, Any]
 
 
+class ServerConfig(BaseModel):
+    port: int = Field(
+        default=8321,
+        description="Port to listen on",
```

Comment:
How about enforcing a specific range for validation? eg: `ge=1024, le=65535`

Reply:
done